documentation/usage/gradle/index.md

```diff
@@ -176,6 +176,32 @@ flyway {
 
 For details on how to setup and use custom Gradle configurations, see the [official Gradle documentation](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.ConfigurationContainer.html).
 
+### Using Flyway Database Types
```

Comment:
Can you reword this to `Adding dependencies on Flyway Database Types` - it would be better to put `dependency` in some form in the header